contracts/pallet-ibc/src/light_clients.rs

```diff
@@ -19,12 +19,20 @@ use ics11_beefy::{
 	client_message::BEEFY_CLIENT_MESSAGE_TYPE_URL, client_state::BEEFY_CLIENT_STATE_TYPE_URL,
 	consensus_state::BEEFY_CONSENSUS_STATE_TYPE_URL,
 };
-use sp_core::{ed25519, H256};
+use sp_core::{crypto::ByteArray, ed25519, H256};
 use sp_runtime::{
 	app_crypto::RuntimePublic,
 	traits::{BlakeTwo256, ConstU32, Header},
 	BoundedBTreeSet, BoundedVec,
 };
+use tendermint::{
+	crypto::{
+		signature::{Error as TendermintCryptoError, Verifier},
+		Sha256,
```

Comment:
don't you want to rename this as TendermintSha256? I'm wondering if this could potentially bring a clash if we'd be using another SHA impl?